src/backend/distributed/executor/multi_real_time_executor.c

```diff
@@ -225,8 +225,7 @@ MultiRealTimeExecute(Job *job)
 	 */
 	if (taskFailed)
 	{
-		ereport(ERROR, (errmsg("failed to execute job " UINT64_FORMAT, job->jobId),
-						errdetail("Failure due to failed task %u", failedTaskId)));
+		ereport(ERROR, (errmsg("failed to execute task %u", failedTaskId)));
```

Comment:
Seems more trouble than it's worth, and it's not a given that these messages will remain the same across different executors.